src/mono/wasm/testassets/WasmBasicTestApp/WasmBasicTestApp.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.WebAssembly">
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
Is there a reason why this project can't use `$(NetCoreAppCurrent)`?

Reply:
We can do that.